src/components/Sum.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export const Sum = (props) => {
+  const { a, b } = props;
+
+  return (
+    <p>{`Sum of ${a} and ${b} is ${(a) + (b)}`}</p>
```

Comment:
I think you don't need to interpolate if you're using a `<p>` tag

Reply:
Okay noted thanks